sjsonnet/test/src-js/sjsonnet/FileTests.scala

```diff
@@ -5,6 +5,14 @@ import scala.scalajs.js
 import utest._
 
 object FileTests extends TestSuite {
+  val skippedTests = Set(
+    "sjsonnet/test/resources/test_suite/dos_line_endings.jsonnet",
+    "sjsonnet/test/resources/test_suite/stdlib.jsonnet",
+    "sjsonnet/test/resources/test_suite/recursive_function.jsonnet",
+    "sjsonnet/test/resources/test_suite/regex.jsonnet",
+    "sjsonnet/test/resources/test_suite/trace.jsonnet"
```

Comment:
Can this be just named with `regex.jsonnet`? as we put the tests in `sjsonnet/test/resources/test_suite/`, we can drop the prefix